Change title and abstract and add seo metadata (#18)
OpenLiberty/draft-guide-kubernetes-microprofile-health: README.adoc

:page-related-guides: ['microprofile-health', 'kubernetes-intro']
:common-includes: https://raw.githubusercontent.com/OpenLiberty/guides-common/master
:source-highlighter: prettify
:page-seo-title: Kubernetes readiness and liveness probe tutorial

> :page-seo-description: How to setup liveness probes and readiness probes in Kubernetes

Reviewer: The SEO title and description should map to what we have in the guide too.  We can build upon what we have for the actual title and abstract.

Reviewer: What we have done for recent guides, which is similar to what you have, isn't ideal, @proubatsis.

> :page-seo-description: How to check microservice health status using MicroProfile Health with Kubernetes liveness probes and readiness probes

Reviewer: ```suggestion
:page-seo-description: A tutorial and example on how to report health status of microservices using MicroProfile Health Check and check them with Kubernetes liveness and readiness probes
```

> :page-seo-description: How to check microservice health status using MicroProfile Health with Kubernetes liveness probes and readiness probes

Reviewer: Do we want to change page-description to "Learn how to check the health of microservices on Kuberenetes by setting up readiness and liveness probes to inspect MicroProfile Health Check endpoints"?

May want to double check how it looks on the All Guides page i.e. in the card for the guide first.

= Checking the health of microservices on Kubernetes

[.hidden]
NOTE: This repository contains the guide documentation source. To view the guide in published form, view it on the https://openliberty.io/guides/{projectid}.html[Open Liberty website].

> Learn how to keep microservices healthy on Kuberenetes by setting up readiness and liveness probes to check MicroProfile Health endpoints.

Reviewer: Should make the abstract consistent with the title too.  I will use some form of "check the health of services".

> Learn how to check a microservice's health status on Kuberenetes by setting up readiness and liveness probes to check MicroProfile Health endpoints.

Reviewer: ```suggestion
Learn how to check the health of microservices on Kuberenetes by setting up readiness and liveness probes to inspect MicroProfile Health Check endpoints.
```
